images/linux/scripts/installers/azpowershell.sh

```diff
@@ -6,92 +6,33 @@
+done
 
 # Run tests to determine that the software installed as expected
 echo "Testing to make sure that script performed as expected, and basic scenarios work"
-if ! pwsh -Command '$actualPSModulePath = $env:PSModulePath ; $env:PSModulePath = "/usr/share/az_1.0.0:" + $env:PSModulePath;
-    if (!(get-module -listavailable -name Az.accounts)) {
-        Write-Host "Az Module was not installed"; $env:PSModulePath = $actualPSModulePath; exit 1
-    }
-    $env:PSModulePath = $actualPSModulePath
-    $actualPSModulePath = $env:PSModulePath ; $env:PSModulePath = "/usr/share/az_1.6.0:" + $env:PSModulePath;
-    if (!(get-module -listavailable -name Az.accounts)) {
-        Write-Host "Az Module was not installed"; $env:PSModulePath = $actualPSModulePath; exit 1
-    }
-    $env:PSModulePath = $actualPSModulePath
-    $actualPSModulePath = $env:PSModulePath ; $env:PSModulePath = "/usr/share/az_2.3.2:" + $env:PSModulePath;
-    if (!(get-module -listavailable -name Az.accounts)) {
-        Write-Host "Az Module was not installed"; $env:PSModulePath = $actualPSModulePath; exit 1
-    }
-    $env:PSModulePath = $actualPSModulePath
-    $actualPSModulePath = $env:PSModulePath ; $env:PSModulePath = "/usr/share/az_2.6.0:" + $env:PSModulePath;
-    if (!(get-module -listavailable -name Az.accounts)) {
-        Write-Host "Az Module was not installed"; $env:PSModulePath = $actualPSModulePath; exit 1
-    }
-    $env:PSModulePath = $actualPSModulePath
-    $actualPSModulePath = $env:PSModulePath ; $env:PSModulePath = "/usr/share/az_2.8.0:" + $env:PSModulePath;
-    if (!(get-module -listavailable -name Az.accounts)) {
-        Write-Host "Az Module was not installed"; $env:PSModulePath = $actualPSModulePath; exit 1
-    }
-    $env:PSModulePath = $actualPSModulePath
-    $actualPSModulePath = $env:PSModulePath ; $env:PSModulePath = "/usr/share/az_3.1.0:" + $env:PSModulePath;
-    if (!(get-module -listavailable -name Az.accounts)) {
-        Write-Host "Az Module was not installed"; $env:PSModulePath = $actualPSModulePath; exit 1
-    }
-    $env:PSModulePath = $actualPSModulePath
-    $actualPSModulePath = $env:PSModulePath ; $env:PSModulePath = "/usr/share/az_3.5.0:" + $env:PSModulePath;
-    if (!(get-module -listavailable -name Az.accounts)) {
-        Write-Host "Az Module was not installed"; $env:PSModulePath = $actualPSModulePath; exit 1
-    }
-    $env:PSModulePath = $actualPSModulePath
-    $actualPSModulePath = $env:PSModulePath ; $env:PSModulePath = "/usr/share/az_3.8.0:" + $env:PSModulePath;
-    if (!(get-module -listavailable -name Az.accounts)) {
-        Write-Host "Az Module was not installed"; $env:PSModulePath = $actualPSModulePath; exit 1
-    }
-    $env:PSModulePath = $actualPSModulePath'; then
-    exit 1
-fi
+for version in ${versions[@]}; do
+    pwsh -Command '
+        $actualPSModulePath = $env:PSModulePath;
+        $env:PSModulePath = "/usr/share/az_${version}:" + $env:PSModulePath;

```

Comment:
${version} won't be resolved properly in single quote

Reply:
Fixed, should be ok now.